include/VocalTractLabApi/VocalTractLabApi.h

```diff
@@ -191,33 +191,52 @@ C_EXPORT int vtlGetTractParams(const char *shapeName, double *tractParams);
 
 
 // ****************************************************************************
-// Exports the vocal tract contours for the given vector of vocal tract
+// Exports the vocal tract contours for the given array of vocal tract
 // parameters as a SVG file (scalable vector graphics).
 //
+// Parameters out:
+// o outTractParams: The bio-mechanically constrained tract parameter array
+// that is actually used to determine the shape.
+//
 // Function return value:
 // 0: success.
 // 1: The API has not been initialized.
 // 2: Writing the SVG file failed.
 // ****************************************************************************
 
-C_EXPORT int vtlExportTractSvg(double *tractParams, const char *fileName);
+
+C_EXPORT int vtlExportTractSvg(double *inTractParams, double *outTractParams, const char *fileName);
 
 
 // ****************************************************************************
-// Provides the tube data (especially the area function) for the given vector
-// of tractParams. The vectors tubeLength_cm, tubeArea_cm2, and tubeArticulator, 
-// must each have as many elements as tube sections.
-// The values incisorPos_cm, tongueTipSideElevation, and velumOpening_cm2 are 
-// one double value each.
+// Provides the tube data (especially the area function) for a given set
+// of supra-glottal parameters.
+// 
+// Parameters in:
+// o inTractParams: Is an array of supra-glottal parameters with 
+//     numVocalTractParams elements.
+// 
+// Parameters out:
+// o outTractParams: Is an array of supra-glottal parameters (biological constraints
+//     applied) with numVocalTractParams elements.
+// o tubeLength_cm: Array containing the lengths of the individual tube sections.
+//     This array must have as many elements as tube sections.
+// o tubeArea_cm2: Array containing the cross-sectional areas of the individual 
+//     tube sections. This array must have as many elements as tube sections.
+// o tubeArticulator: Array containing the lengths of the individual tube sections.
+//     This array must have as many elements as tube sections.
+// o incisorPos_cm: A double value.
+// o tongueTipSideElevation: A double value.
+// o velumOpening_cm2: A double value.
 //
 // Function return value:
 // 0: success.
 // 1: The API has not been initialized.
 // ****************************************************************************
 
-C_EXPORT int vtlTractToTube(double* tractParams,
-  double* tubeLength_cm, double* tubeArea_cm2, int* tubeArticulator,
-  double* incisorPos_cm, double* tongueTipSideElevation, double* velumOpening_cm2);
+C_EXPORT int vtlTractToTube(double *inTractParams, double *outTracParams,
```

Comment:
You mentioned that we decided on using the `type *name` notation for pointers, but I don't remember that discussion and both versions can be found in the legacy codebase. When did we settle that?

Because I would in fact prefer the other way. The type is `pointer to X`, not `X`, so writing it as `X* name` makes so much more sense to me. Not trying to be nit-picky, just curious when we decided to do it differently.